foundation/foundation-mda/src/main/resources/profiles.json

```diff
@@ -350,9 +350,6 @@
 			},
 			{
 				"name": "dataAccessDockerPomFile"
-			},
-			{
-				"name": "vaultDockerPomFile"
```

Comment:
I: Disable vault inclusion by default, since UC isn't ready to be rolled out yet.